### PR TITLE
CORTX-29502: PI-6 OVA Instructions and Release Notes

### PR DESCRIPTION
**Pull request by venkuppu-chn**
### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [ ] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [ ] tested locally
 - [ ] added new dependencies
 - [ ] updated the docs
 - [ ] added a test

-----
[View rendered doc/ova/2.0.0/PI-6/CHANGELOG.md](https://github.com/venkuppu-chn/cortx/blob/main/doc/ova/2.0.0/PI-6/CHANGELOG.md)
[View rendered doc/ova/2.0.0/PI-6/S3_IO_Operations.md](https://github.com/venkuppu-chn/cortx/blob/main/doc/ova/2.0.0/PI-6/S3_IO_Operations.md)

### CODE REVIEW

**Review by shailesh-vaidya — COMMENTED** [review]
- Do we need to add the below steps to use `s3.seagate.com` for IO operations from https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-5/S3_IO_Operations.md#procedure 

Edit /etc/hosts file and copy the IP of service data pod i.e. cortx-data-pod on same OVA image and add following entry if it does not exist
```
<IP_Address_of_cortx-data-pod> s3.seagate.com iam.seagate.com
```

- Do we need to add steps to test IO operations from outside OVA VM

**Comment by venkuppu-chn** [comment]
> * Do we need to add the below steps to use `s3.seagate.com` for IO operations from https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-5/S3_IO_Operations.md#procedure
> 
> Edit /etc/hosts file and copy the IP of service data pod i.e. cortx-data-pod on same OVA image and add following entry if it does not exist
> 
> ```
> <IP_Address_of_cortx-data-pod> s3.seagate.com iam.seagate.com
> ```
> 
> * Do we need to add steps to test IO operations from outside OVA VM

@kupranay and @cdeshmukh Do we need to add the steps mentioned by Shailesh? Kindly confirm.

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-5.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
+
```
Comment (mukul-seagate11) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-5.rst:1]: @venkuppu-chn, change file path from CORTX_on_Open_Virtual_Appliance_PI-5.rst to CORTX_on_Open_Virtual_Appliance_PI-6.rst

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

**Comment by kupranay** [comment]
> https://github.com/Seagate/cortx/pull/1476#issuecomment-1082777909

@venkuppu-chn This should not be required

**Comment by johnbent** [comment]
@venkuppu-chn , why is this labeled 'Do not merge'? Will it eventually be merged?

**Comment by mukul-seagate11** [comment]
> @venkuppu-chn , why is this labeled 'Do not merge'? Will it eventually be merged?

@johnbent, Its because we are waiting for the QA with final validation report so once good, it will be merged

**Comment by johnbent** [comment]
> > @venkuppu-chn , why is this labeled 'Do not merge'? Will it eventually be merged?
> 
> @johnbent, Its because we are waiting for the QA with final validation report so once good, it will be merged

Awesome! Thanks. By the way, Greg Kittlison shared an article with me today about best practices for github and it suggested opening pull requests early just like you are doing cool. Nice work!

https://petabridge.com/blog/use-github-professionally/

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-5.rst (hunk 1 above)
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-5.rst]: Done. Thanks Mukul.

**Comment by mukul-seagate11** [comment]
@shailesh-vaidya @kupranay @cdeshmukh, can you review & approve the instructed updated for OVA

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+  - RAM: 8GB
+  - Processor: 4
+  - NIC: 1
+  - OS Disk: 1 disk of 20GB
+  - Data Disks: 3 disks of 5GB each
+
+**Prerequisites:**
+
+- Download the `CORTX OVA <https://cortxova.s3.us-west-2.amazonaws.com/cortx-k8s-0.0.16.ova>`_ from `our release page <https://github.com/Seagate/cortx/releases/latest>`_.
```
Comment (mukul-seagate11) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:30]: Updated OVA image link will be updated once available

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+`VMware Workstation <https://www.vmware.com/products/workstation-pro.html>`_,
+`VMware Fusion <https://www.vmware.com/in/products/fusion/fusion-evaluation.html>`_
+
+**Important**: If you are running the VM in any of the VMWare hypervisors, it is not recommended to use VMware Tools, as CORTX may break due to kernel dependencies. For the same reason, please do not update the operating system in the image as that also might cause it to fail.
+
+**Specifications:**
+
+- CORTX OVA image is created with following minimum configuration:
+
+  - RAM: 8GB
```
Comment (mukul-seagate11) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:22]: Update the spec as per new OVA as,

- RAM: 10GB
- Processor: 8

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst (hunk 3 above)
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst]: Done.

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+#. To perform the S3 IO operations, refer the instructions in `S3 IO operations <https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-6/S3_IO_Operations.md>`_.
+
+#. BOOM. You're all done and you're AWESOME. 
+
+   Thanks for checking out the CORTX system; we hope you liked it. Hopefully you'll stick around and participate in our community and help make it even better.
+
+ 
+
+***************
+Troubleshooting
```
Comment (mukul-seagate11) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:71]: remove Troubleshooting section as not required for now

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

**Comment by mukul-seagate11** [comment]
Also create a new section for CSM API validation

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+   **Note** Wait for 5 - 10 mins till the cluster services are started.
+   
+#. Run the following command to check the health of CORTX using hctl by running this command:
+
+   ::
+
+       hctl status
+
+ 
+#. To perform the S3 IO operations, refer the instructions in `S3 IO operations <https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-6/S3_IO_Operations.md>`_.
```
Comment (mukul-seagate11) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:62]: Update the correct URL for S3 IO operations as the link is broken

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  - RAM: 10GB
+  - Processor: 8
+  - NIC: 1
+  - OS Disk: 1 disk of 20GB
+  - Data Disks: 3 disks of 5GB each
+
+**Prerequisites:**
+
+- Download the `CORTX OVA <https://cortxova.s3.us-west-2.amazonaws.com/ova-2.0.0-713.ova>`_ from `our release page <https://github.com/Seagate/cortx/releases/latest>`_.
+- To perform the S3 IO operations,refer to these instructions in `S3 IO Operations <https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-6/S3_IO_Operations.md>`_.
```
Comment (mukul-seagate11) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:31]: @venkatesh, Update the correct URL for S3 IO operations as the link is broken

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+  - RAM: 10GB
+  - Processor: 8
+  - NIC: 1
+  - OS Disk: 1 disk of 20GB
+  - Data Disks: 3 disks of 5GB each
+
+**Prerequisites:**
+
+- Download the `CORTX OVA <https://cortxova.s3.us-west-2.amazonaws.com/ova-2.0.0-713.ova>`_ from `our release page <https://github.com/Seagate/cortx/releases/latest>`_.
```
Comment (cdeshmukh) [inline doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:30]: https://github.com/Seagate/cortx/releases/tag/2.0.0-585
link has legacy cortx-s3server reference still, should be replaced with cortx-rgw (cc @shailesh-vaidya to confirm that tagging is added for this repo too)

> To clone & checkout from tag 2.0.0-585 where following repos have the CORTX tagging feature:
cortx-s3server

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst (hunk 5 above)
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:62]: What is the correct URL? This link will work once PR is merged if I'm not wrong.

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst (hunk 6 above)
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:31]: @mukul-seagate11 What is the correct URL? I believe this link will work once PR is merged.

Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:31]: I see as its a newly created file so will be accessible once PR is merged so you can ignore

File: doc/ova/2.0.0/PI-6/CHANGELOG.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+## Bug Fixes
+
+- Restructure CSM-S3 integration code[CORTX-28035]
+- [RGW] REST API to modify IAM user [CORTX-28042]
+- Support bundle enhancement for adding time filter [EOS-28031]
+- Update CSM to work on Rocky Linux 8.2 [EOS-28018]
+- Remove dependency on LDAP for user management [EOS-28020]
+- Integrate Shutdown Signaling REST API with HA [EOS-25532]
+- Manage user not able to edit self email id [EOS-26602]
+- Support Bundle filter implementation [EOS-27251]
```
[the remark this answers is not in the document]
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CHANGELOG.md:13]: Removed.

File: doc/ova/2.0.0/PI-6/CHANGELOG.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+- fdmi UT fixes [EOS-27220]
+- Update version for each component in bootstrap phase [EOS-28599]
+- remove 3rd party mini provisioner calls [CORTX-28645]
+
+
+## New Features
+
+- Improvement in aux pool enable/disable implementation [EOS-25537]
+- Improve http command for hctl status [EOS-27236]
+- [F-53B]Optimize component bundle size - Motr [EOS-25683]
```
[the remark this answers is not in the document]
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CHANGELOG.md:43]: Removed the Feature ID.

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst (hunk 4 above)
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst]: Done

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst (hunk 7 above)
Comment (venkuppu-chn) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:30]: @mukul-seagate11 Please help me to understand about the @chaozhang95 comments.

Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:30]: @cdeshmukh, https://github.com/Seagate/cortx/releases/tag/2.0.0-585 visible as old release but the instructions in this page will be updated with new release build i.e. 2.0.0-713.ova in the release page before releasing

**Review by mukul-seagate11 — APPROVED** [review]
looks good

File: doc/ova/2.0.0/PI-6/CHANGELOG.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (cdeshmukh) [reply doc/ova/2.0.0/PI-6/CHANGELOG.md:12]: i said merge, keep both tickets in list

File: doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst (hunk 7 above)
Comment (cdeshmukh) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:30]: so page with new release build i.e. 2.0.0-713.ova will not have s3server reference ? if yes, then its fine

Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst:30]: yes correct, s3server refs are removed in 2.0.0-713.ova

File: doc/ova/2.0.0/PI-6/CHANGELOG.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (mukul-seagate11) [reply doc/ova/2.0.0/PI-6/CHANGELOG.md]: @cdeshmukh, can you review & approve the PR

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**